docs/reference/index.md

```diff
@@ -3,26 +3,39 @@ title: "Uport Connect"
 index: 0
 category: "reference"
 type: "content"
+source: "https://github.com/uport-project/uport-connect/blob/develop/docs/reference/index.md"
 ---
 
+
+
+## Classes
+
+<dl>
+<dt><a href="#Connect">Connect</a></dt>
+<dd></dd>
+<dt><a href="#UportSubprovider">UportSubprovider</a></dt>
+<dd><p>A web3 style provider which can easily be wrapped with uPort functionality.
+ Builds on a base provider. Used in Connect to wrap a provider with uPort specific
+ functionality.</p>
+</dd>
+</dl>
+
 <a name="Connect"></a>
 
 ## Connect
 **Kind**: global class  
 
 * [Connect](#Connect)
```

Comment:
Maybe make the ordering of methods here match the `usage.md`?

Reply:
same as above, if can't change auto generation easily, not worth changing manually at the moment